src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -35,8 +36,10 @@ public BlazorClient()
             });
         }
 
-        public TimeSpan? DefaultConnectionTimeout { get; set; } = TimeSpan.FromSeconds(20);
-        public TimeSpan? DefaultOperationTimeout { get; set; } = TimeSpan.FromMilliseconds(500);
+        public TimeSpan? DefaultConnectionTimeout { get; set; } = Debugger.IsAttached ?
```

Comment:
Why is the timeout different between debugger attached and not?

Unrelated: why is this nullable?

Reply:
It was annoying to have these timeouts kick in while I was debugging the app. We do this sort of thing in other places where timeouts are shut off if you're debugging.

Reply:
I didn't super dig in to why it was null. But does seem like something that could be changed.